CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+### How to contribute:
+
+It's recommended that you first contact ctsm-software@ucar.edu to discuss the changes or additions that you propose.
+Work with them to think about the design and implemention of the changes as well as the planning
+and scheduling for the change to come into CTSM.
+
+It's also recommened that you join the ctsm-dev@ucar.edu google groups in order 
+to be informed about the science that is happening in CTSM as well as the latest develepments and tags.
+The ctsm-dev group is here:
+
+https://groups.google.com/a/ucar.edu/forum/#!forum/ctsm-dev
+
+Use the help from the wiki below to setup a fork and personal branch in GitHub to put your developments
+on and keep up to date with the master branch of CTSM. Once the changes are sufficiently advanced you
+can form a Pull Request on GitHub. Either from your fork on GitHub, or from the main ESCOMP GitHub page
+for CTSM (be sure to hit the "compare across forks" link at the top of the page when you first create
+the pull request).
+
+https://github.com/ESCOMP/ctsm/pulls
+
+This allows you to show your proposed changes and start getting feedback on them (even if they aren't finished). 
+This also allows your changes to be planned for and slated for a time to come into CTSM master. In most 
+cases you won't merge the Pull Request yourself, but a software engineer responsible for CTSM will do 
+additional testing and bring the changes to CTSM master.
+
+# CTSM Developers Guide:
```

Comment:
(I thought I made this comment in my first review, but I don't see it anywhere, so I guess I never submitted it.)

I don't think we need a heading for each link. Could this instead look like this?:

```markdown
# Helpful resources

- CTSM Developers Guide: URL

- CTSM Wiki: URL

- etc.
```

But maybe I'll be more okay with headers once they're fixed to be smaller (see the above comment about header levels).